### PR TITLE
Software Composition Analysis (SCA)

### PR DESCRIPTION
**Pull request by fpintoppb**

### CODE REVIEW

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -28,11 +28,11 @@ jobs:
     runs-on: ubuntu-latest
     services:
       mysql:
-        image: mysql:8
+        image: mysql:8.0.30
         ports:
         - 8877:3306
         # needed because the container does not provide a healthcheck
-        options: --health-cmd "mysqladmin ping" --health-interval 10s --health-timeout 5s --health-retries=5 -e MYSQL_ROOT_PASSWORD=root --entrypoint sh mysql -c "exec docker-entrypoint.sh mysqld --default-authentication-plugin=mysql_native_password"
+        options: --health-cmd "mysqladmin ping" --health-interval 10s --health-timeout 5s --health-retries=5 -e MYSQL_ROOT_PASSWORD=root --entrypoint sh mysql:8.0.30 -c "exec docker-entrypoint.sh mysqld --default-authentication-plugin=mysql_native_password"
```
Comment (DDuarte) [inline .github/workflows/run_tests.yml:35]: this `:8.0.30` does not look right

File: .github/workflows/run_tests.yml [hunk 2]
```diff
@@ -61,11 +61,11 @@ jobs:
 
     services:
       mysql:
-        image: mysql:8
+        image: mysql:8.0.30
         ports:
         - 8877:3306
         # needed because the container does not provide a healthcheck
-        options: --health-cmd "mysqladmin ping" --health-interval 10s --health-timeout 5s --health-retries=5 -e MYSQL_ROOT_PASSWORD=root --entrypoint sh mysql -c "exec docker-entrypoint.sh mysqld --default-authentication-plugin=mysql_native_password"
+        options: --health-cmd "mysqladmin ping" --health-interval 10s --health-timeout 5s --health-retries=5 -e MYSQL_ROOT_PASSWORD=root --entrypoint sh mysql:8.0.30 -c "exec docker-entrypoint.sh mysqld --default-authentication-plugin=mysql_native_password"
```
Comment (DDuarte) [inline .github/workflows/run_tests.yml:68]: same as above

File: surface/pytest.ini [hunk 3]
```diff
@@ -3,5 +3,5 @@ DJANGO_SETTINGS_MODULE = surface.settings_test
 python_files = tests.py test_*.py
 junit_family = xunit2
 # add --create-db to your command line when re-creating DB is required, no need to remove reuse-db from here
-addopts = --reuse-db --disable-socket --allow-unix-socket --allow-hosts=127.0.0.1
+addopts = --reuse-db
```
Comment (DDuarte) [inline surface/pytest.ini:6]: you sure?

File: surface/sca/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+
+### Usage
+
+#### Synchronize SBOMs
+To synchronize SBOMs across your projects, run: \
+`python manage.py resync_sbom_repo`
+
+#### Check for public dependencies
+In order to mark a dependency as internal or public we use multiple repoistories to check it's existence \
```
Comment (DDuarte) [inline surface/sca/README.md:33]:
```suggestion
In order to mark a dependency as internal or public we use multiple repositories to check it's existence \
```

File: surface/pytest.ini (hunk 3 above)
Comment (fpintoppb) [reply surface/pytest.ini:6]: local testing

**Review by DDuarte — APPROVED** [review]

**Pull request merged**